cleanlab_studio/internal/settings.py

```diff
@@ -73,10 +78,13 @@ def update_version(self) -> None:
         self.save()
 
     def validate_version(self) -> None:
-        if semver.compare(MIN_SETTINGS_VERSION, self.version) == 1:
+        min_version = semver.VersionInfo.parse(MIN_SETTINGS_VERSION)
+        max_version = semver.VersionInfo.parse(MAX_SETTINGS_VERSION)
+        current_version = semver.VersionInfo.parse(self.version)
+        if current_version < min_version:
             # TODO add proper settings migrations
             raise SettingsError("Settings file must be migrated or re-generated.")
-        elif semver.compare(MAX_SETTINGS_VERSION, self.version) == -1:
+        elif current_version > max_version:
```

Comment:
Curious how this change relates to the rest of the PR?

Reply:
It was throwing errors in the CI otherwise